packages/navigator/src/browser/navigator-widget.ts

```diff
@@ -81,13 +83,11 @@ export class FileNavigatorWidget extends FileTreeWidget {
     }
 
     protected handleCopy(event: ClipboardEvent): void {
-        const node = this.model.selectedFileStatNode;
-        if (!node) {
-            return;
+        const uris = [...this.model.selectedFileStatNodes].map(node => node.uri.toString());
```

Comment:
copying?

Reply:
A leftover before we had the selection as an Array.